l10n_es_aeat_mod347/mod347.py

```diff
@@ -25,6 +25,17 @@ class L10nEsAeatMod347Report(orm.Model):
     _name = "l10n.es.aeat.mod347.report"
     _description = "AEAT 347 Report"
 
+    def btn_list_records(self, cr, uid, ids, context=None):
+        value = {
```

Comment:
Sí, la verdad es que lo que comentas es un añadido que puede ser útil en estos casos. Creo que podemos dejarlo puesto que no molesta nada, pero renómbralo a "Open list in new window", que expresa mejor lo que hace.